pkg/archive/archive_test.go

```diff
@@ -153,9 +160,22 @@ func TestCompressStreamBzip2Unsupported(t *testing.T) {
 	}
 	defer dest.Close()
 
-	_, err = CompressStream(dest, Xz)
+	_, err = CompressStream(dest, Bzip2)
```

Comment:
Is this change necessary for this PR?

Reply:
Not necessary for this PR, but a drive-by change since I was adding more tests. I can add it in to a separate PR if that's preferred.

Reply:
Please do. CC me and we can get that clean up merged!